pkgs/os-specific/linux/alp/default.nix

```diff
@@ -0,0 +1,61 @@
+# This file was generated by GoReleaser. DO NOT EDIT.
```

Comment:
This is a machine-generated file. We will not accept it.

Reply:
It would have been convenient to reuse the goreleaser output. However, I understand that you do not want to merge something that says "do not edit".

The old code also had further issues like it referenced only some remote hosted pre-built artifacts (from go releaser). I see that the nix philosophy prefers to have the whole derivation process. I therefore used the buildGoModule as currently recommended in the newly pushed code.

I therefore hope this has a chance of becoming accepted.

However, it is still in draft state and I still need to complete the unticket checkboxes in the description (hopefully tomorrow).